posthog/email.py

```diff
@@ -97,7 +104,13 @@ def _send_email(
 
 class EmailMessage:
     def __init__(
-        self, campaign_key: str, subject: str, template_name: str, template_context: Optional[Dict] = None,
+        self,
+        campaign_key: str,
+        subject: str,
+        template_name: str,
+        template_context: Optional[Dict] = None,
+        headers: Optional[Dict] = None,
+        reply_to: str = "",
```

Comment:
Nit: lack of `reply_to` should be represented everywhere by `None` instead of an empty `str`.

Reply:
Well at least in models I think is best practice to use an empty string as the _null/nil_ state. Otherwise you're left guessing which is the actual null/nil state, but I think it's fine to do this here.